src/polygon2d.jl

```diff
@@ -0,0 +1,30 @@
+"""
+```
+draw_polygon_perimeter!(img, vertices)
```

Comment:
Since `draw` is in the name of the package, let's leave it out of the name of the function.